src/runtime/runtime_wasip1.go

```diff
@@ -149,3 +149,22 @@ type (
 		}
 	}
 )
+
+// Need to stub these for `os/signal`, but wasi doesn't support signals.
+
+//go:linkname signal_disable os/signal.signal_disable
+func signal_disable(uint32) {}
+
+//go:linkname signal_enable os/signal.signal_enable
+func signal_enable(uint32) {}
+
+//go:linkname signal_ignore os/signal.signal_ignore
+
```

Comment:
Nit: the use of newlines between `go:linkname`s and their functions is inconsistent.